test/widgets/channel_colors_checks.dart

```diff
@@ -3,7 +3,7 @@ import 'dart:ui';
 import 'package:checks/checks.dart';
 import 'package:zulip/widgets/channel_colors.dart';
 
-extension StreamColorSwatchChecks on Subject<ChannelColorSwatch> {
+extension ChannelColorSwatchChecks on Subject<ChannelColorSwatch> {
```

Comment:
I think this will be better included in [`88dd4b1`](88dd4b18a46b9a82a195b17d03813d8945cd83a9): channel_colors [nfc]: Rename StreamColorSwatch to ChannelColorSwatch.